### PR TITLE
Add evaluation plan for the closed-loop evaluation

### PR DESCRIPTION
**Pull request by perone**
This PR will add the evaluation plan (`EvaluationPlan`) and its unit-testing as well as composite metrics and validators protocols.

### CODE REVIEW

**Comment by perone** [comment]
PTAL @lucabergamini, I fixed the Py 3.7 issues.

File: l5kit/setup.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 from l5kit import __version__
 
+
```
Comment (lucabergamini) [inline l5kit/setup.py:6]: isort should not run on this file tbh but I'm fine with this change

File: l5kit/l5kit/cle/closed_loop_evaluator.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+import torch
+
+from l5kit.cle.composite_metrics import SupportsCompositeMetricCompute
+from l5kit.cle.metrics import SupportsMetricCompute
+from l5kit.cle.validators import SupportsMetricValidate, ValidatorOutput
+from l5kit.simulation.unroll import SimulationOutput
+
+
+class EvaluationPlan:
```
Comment (lucabergamini) [inline l5kit/l5kit/cle/closed_loop_evaluator.py:11]: just to check: this doesn't have a method to:
- evaluate metrics
- evaluate validators
- evaluate composed metrics
 right?

that will come later somewhere else

**Review by lucabergamini — APPROVED** [review]
LGTM

File: l5kit/l5kit/cle/closed_loop_evaluator.py (hunk 2 above)
Comment (perone) [reply l5kit/l5kit/cle/closed_loop_evaluator.py:11]: Correct, that will come from the `ClosedLoopEvaluator`

**Pull request merged**